cli/src/installers/drizzle.ts

```diff
@@ -30,7 +30,7 @@ export const drizzleInstaller: Installer = ({
         {
           planetscale: "@planetscale/database",
           mysql: "mysql2",
-          postgres: "postgres",
```

Comment:
We want to use the Postgres packagedor the actual querying etc. I believe pg might be a peer dependency for drizzle kit? Then we'd need to add pg as a dev dependency whilst keeping Postgres as the main production dependency